pulse2percept/models/beyeler2019.py

```diff
@@ -961,3 +1034,32 @@ def predict_percept(self, implant, t_percept=None):
                                  f"built for {self.spatial.eye}.")
         return super(AxonMapModel, self).predict_percept(implant,
                                                          t_percept=t_percept)
+
+    def __deepcopy__(self, memodict={}):
```

Comment:
It seems that this is the common copy format for all `Model` objects. Perhaps we could implement this once in `Model` and not have to reimplement it in subclasses?